localstack/services/cloudformation/engine/transformers.py

```diff
@@ -53,7 +58,9 @@ def _visit(obj, **_):
             if transformer_class:
                 transformer = transformer_class()
                 parameters = transform.get("Parameters") or {}
-                parameters = resolve_refs_recursively(stack_name, resources, mappings, parameters)
+                parameters = resolve_refs_recursively(
```

Comment:
Not a huge fan of adding more parameters to this function. If it needs it then so be it, and it's probably worth it to remove anything that isn't a resource from the `resources` dict.

Reply:
Yeah it's more of a temporary measure anyway, even if that temporary is a span of a few months. Just helps in being more explicit in what the specific parts of the engine need in terms of parameters, resources, etc.